modules/eco-self-node-remediation-operator-about.adoc

```diff
@@ -44,13 +60,14 @@ spec:
 +
 If a watchdog device is unavailable, the `SelfNodeRemediationConfig` CR uses a software reboot.
 <3> Specify if you want to enable software reboot of the unhealthy nodes. By default, the value of `isSoftwareRebootEnabled` is set to `true`. To disable the software reboot, set the parameter value to `false`.
-<4> Specify the timeout duration to check connectivity with each API server. When this duration elapses, the Operator starts remediation.
-<5> Specify the frequency to check connectivity with each API server.
-<6> Specify a threshold value. After reaching this threshold, the node starts contacting its peers.
-<7> Specify the timeout duration for the peer to connect the API server.
-<8> Specify the timeout duration for establishing connection with the peer.
-<9> Specify the timeout duration to get a response from the peer.
-<10> Specify the frequency to update peer information, such as IP address.
+<4> Specify the timeout duration to check connectivity with each API server. When this duration elapses, the Operator starts remediation. The timeout duration must be more than or equal to 10ms.
```

Comment:
If you feel like making an out of scope improvement, this should be `10ms` (the [literal value](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference), 10 ms (ISG wants a space here), or 10 milliseconds (to match the rest of the callouts). (If we had all the time in the world, I'd suggest making all of the values in the callouts use one of the first two formats.)

Reply:
Changed it to 10 milliseconds